pkgs/by-name/ma/magma/package.nix

```diff
@@ -176,6 +176,10 @@ stdenv.mkDerivation (finalAttrs: {
     ]
   );
 
+  env.CFLAGS = "-DADD_" + lib.optionalString rocmSupport " -fopenmp";
```

Comment:
Moved these from cmake flags to env vars because the current implementation of cmakeFlags doesn't support spaces in the flag value.